src/Symfony/Component/Validator/Constraints/AbstractComparison.php

```diff
@@ -46,7 +47,7 @@ public function __construct($options = null)
             }
 
             if (isset($options['propertyPath']) && !class_exists(PropertyAccess::class)) {
-                throw new ConstraintDefinitionException(sprintf('The "%s" constraint requires the Symfony PropertyAccess component to use the "propertyPath" option.', \get_class($this)));
+                throw new LogicException(sprintf('The "%s" constraint requires the Symfony PropertyAccess component to use the "propertyPath" option.', \get_class($this)));
```

Comment:
No tests to adjust? 🧐

Reply:
Well, tests are green, no ? 😄 

More seriously, it appears that the constructors of the comparison constraints are not tested atm.